src/index.js

```diff
@@ -4,24 +4,29 @@ const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
 const storage = require('./statuscakedata');
 
-const regex = /\s?([-])\s?/;
+const regexp = config.statuscake.regex;
 
-const getTagsFromTitle = Title => Title.split(regex);
+const getTagsFromTitle = title => title.match(new RegExp(regexp));
+const tags = [
+  (config.statuscake.customer),
+  (config.statuscake.service),
+  (config.statuscake.device),
+  (config.statuscake.version)];
 
 const loadtimeGauge = new client.Gauge({
-  name: 'frontend_pagespeed_loadtime_seconds',
+  name: (config.statuscake.nameofloadtimegauge),
   help: 'Loadtime of your page',
-  labelNames: ['customer', 'service', 'device', 'version'],
+  labelNames: (tags),
 });
 const filesizeGauge = new client.Gauge({
-  name: 'frontend_pagespeed_filesize_bytes',
+  name: (config.statuscake.nameoffilesizegauge),
   help: 'Filesize of your page',
-  labelNames: ['customer', 'service', 'device', 'version'],
+  labelNames: (tags),
```

Comment:
Putting `()` arround variables in JS doesn't mean something. 
We should avoid that 🚥

Reply:
🤜